source/Government.cpp

```diff
@@ -136,6 +136,15 @@ void Government::Load(const DataNode &node)
 			raidFleet = GameData::Fleets().Get(child.Token(1));
 		else if(child.Token(0) == "provoked on scan")
 			provokedOnScan = true;
+		else if(child.Token(0) == "restricted")
```

Comment:
Determine if we're in add or remove mode once at the beginning of parsing the node, and then here check that the key is `"restricted"`

(In other words, use the same pattern as other files that support add and remove directives.)

Reply:
> Determine if we're in add or remove mode once at the beginning of parsing the node, and then here check that the key is `"restricted"`
> 
> (In other words, use the same pattern as other files that support add and remove directives.)

I made it similar to how other load function look, and I added the possibility to remove the provoked on scan, the other can be "removed" by putting a default number I think but if they can't I don't know if I should add that within this pr?
like maybe raid, "death sentence", and "display name" should be removable too?